server/master/src/main/java/org/apache/accumulo/master/FateServiceHandler.java

```diff
@@ -155,8 +155,8 @@ public void executeFateOperation(TInfo tinfo, TCredentials c, long opid, FateOpe
             splitFile = createSplitFile(opid, arguments, splitCount, SPLIT_OFFSET);
           } catch (IOException e) {
```

Comment:
It seems like the exception e is just lost.   If not done elsewhere, would be good to log something about e for debugging purposes.

Reply:
Updated to catch and throw exception if method fails.